### ISSUE TITLE
1.4.0 no longer lists azurite under local storage accounts

### ISSUE THREAD

**Issue by xt0rted** [Microsoft/AzureStorageExplorer#576]
**Storage Explorer Version**: 1.4.0
**Platform/OS Version**: Windows 10
**Architecture**: ia32
**Regression From**: 1.3.0?

#### Steps to Reproduce: ####
1. Startup Azurite
2. Open storage explorer
3. Navigate to `Local & Attached` > `Storage Accounts`

#### Expected Experience: ####
Azurite used to show up here and did 5 minutes ago before updating

#### Actual Experience: ####
Azurite no longer shows up so I need to roll back to the previous version of storage explorer but I have no idea how to since the code isn't hosted on GitHub and there's no download links to specific versions 😕

**Comment by xt0rted** [comment]
After poking around I found the 3rd menu item on the left to add a new connection, that lets me connect to azurite again but for some reason my queues, which had test data I was working with, are gone now. Blobs and tables are stored alongside my project (the reason I switched to azurite) are queues not stored there as well?

**Comment by MRayermannMSFT** [comment]
Hey @xt0rted . Yes, emulators are no longer a special case of Local & Attached. Instead there is an option to attach an emulator in the connect dialog (as you saw). As for why you're not seeing queues, that's a good question. Our Azurite expert is out today and tomorrow, but I can try to find some time to investigate. As for rolling back, this link points to the previous version: https://go.microsoft.com/fwlink/?LinkId=809306&clcid=0x409. You always find that link at the bottom of our product page: https://azure.microsoft.com/en-us/features/storage-explorer/

**Comment by xt0rted** [comment]
Now that I see how this works I like the new setup better. I have 2 sites I use Azurite with so it'll be nice being able to use different ports and connect to them at the same time.

**Comment by MRayermannMSFT** [comment]
Yay! Glad you like the change. Apologies that it was a little hard to figure out. We'll use this as a learning experience for the next time we make a breaking/disruptive change like that.

**Comment by MRayermannMSFT** [comment]
Hey @xt0rted , were you ever able to see your queues node? I tried out out today and had no problems seeing it:
![image](https://user-images.githubusercontent.com/21112697/44062580-cfec7614-9f11-11e8-9d30-19d57b99a424.png)

**Comment by MRayermannMSFT** [comment]
My connection string is as follows:
AccountName=devstoreaccount1;AccountKey=**********;DefaultEndpointsProtocol=http;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;

If you're not seeing queues, maybe make sure the QueueEndpoint is in the connection string?

**Comment by xt0rted** [comment]
The node shows but it was empty. Creating new queues works and they persist when closing & reopening the storage explorer.

What seems to be happening is when I stop Azurite (it's being run with Docker) any queues I have are lost. I think they're stored on the Docker container and get cleared when it's stopped and restarted where as blobs & tables are stored alongside my project in a `.azurite` folder.

This has probably been an issue with Azurite all along but I only just noticed it. After updating the storage explorer Azurite wasn't showing so I stopped & started it to see if that helped, and I guess that's when the queues were wiped and I thought it was a storage explorer issue.
